docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<p>same thing as bi code block above. Can use the selected unit indices here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>